src/components/app/Home.css

```diff
@@ -14,6 +14,7 @@
 
 .homeDrop {
   position: absolute;
+  z-index: 10;
```

Comment:
So our general policy is to only increment and decrement the z-index by values of 1, and then document why this is needed. Stacking contexts can be weird to figure out, let me know if you need help figuring this out. I would prefer a solution without a z-index change if possible. Usually changing the ordering of elements, or ensuring the parent element starts a stacking context (for instance by making it `position: relative` or `position: absolute` is enough. I haven't looked at the specifics here though.